RecoVertex/BeamSpotProducer/python/BeamSpotOnline_cfi.py

```diff
@@ -3,8 +3,9 @@
 onlineBeamSpotProducer = _mod.beamSpotOnlineProducer.clone(
                                         src = 'scalersRawToDigi',
                                         setSigmaZ = -1, #negative value disables it.
-                                        gtEvmLabel = 'gtEvmDigis'
+                                        gtEvmLabel = 'gtEvmDigis',
+                                        useBSOnlineRecords = True,
+                                        timeThreshold = 48,
+                                        sigmaZThreshold = 2,
+                                        sigmaXYThreshold = 4
```

Comment:
this should be encapsulated in a `run3_common` modifier (as it was done before for `useTransientRecord`).

Reply:
fixed